collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -1989,98 +1996,59 @@ private void createQuitDialog() {
             }
         }
 
-        String[] items;
+        IconMenuItem[] items;
         if (adminPreferences.getBoolean(AdminKeys.KEY_SAVE_MID,
                 true)) {
-            items = new String[]{getString(R.string.keep_changes),
-                    getString(R.string.do_not_save)};
+            items = new IconMenuItem[]{new IconMenuItem(R.drawable.ic_save_grey_32dp_wrapped, R.string.keep_changes),
```

Comment:
I think its best to use Lists instead of Arrays whenever possible. More flexibility, more compile time safety.
Fastest way to change this would be with an ImmutableList.of() call, but I'll leave that decision up to you.

Reply:
Probably it's premature optimisation from my side, I use Arrays whenever possible for GC reason, but feel it's irrelevant in this case because it very minor thing and devices now are much more performant then 5 years ago.